traverse/ppx_traverse.ml

```diff
@@ -391,7 +391,7 @@ let erase_type_variables = object
   method! core_type core_type =
     let open Viewer in
     match%view core_type with
-    | Ptyp_var _ [@view? {ptyp_loc = loc; _}] -> Builder.ptyp_any ~loc
+    | Tvar _ [@view? {ptyp_loc = loc; _}] -> Builder.ptyp_any ~loc
```

Comment:
Lines like this should be updated to use Ptyp_var and the proper fields, rather than using Tvar and an attribute.

Reply:
I kept the `@view` patterns because it was shorter but I'm happy to switch to the full pattern if you prefer.